config/default/jenkins-infra.yaml

```
@@ -551,6 +551,10 @@ jenkins:
                   - "Job/Build:authenticated"
                   - "Job/ExtendedRead:authenticated"
         system-settings: |
+          unclassified:
+            defaultFolderConfiguration:
+              healthMetrics:
```

have you tested the above as working? I would've expected empty list to be required, but maybe null is fine?
```suggestion
              healthMetrics: []
```

---

I tested successfully, on a local instance, that: 
- It did not crash (to avoid breaking infra.ci as I did yesterday 😅)
- If I add the default health metric through the UI and reload jenkins, then the settings goes back to empty on the UI (because of Casc).

I understand that JCasc is smart and cast a null value to the "empty" array, but I like explicit thing: let me test and add your suggestion.

Thanks!

---

Validated, sounds good.